common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -9,24 +9,18 @@ import Scalaz._
 import com.gu.mediaservice.model.{StaffPhotographer, ContractPhotographer, Photographer}
 
 object PhotographersList {
-  type Store = Map[String, String]
   type CreditBylineMap = Map[String, List[String]]
 
-  import MetadataConfig.{ staffPhotographers, contractedPhotographers }
+  import MetadataConfig.{ Store, staffPhotographers, contractedPhotographers }
 
   def creditBylineMap(store: Store): CreditBylineMap = store
-      .groupBy{ case (photographer, publication) => publication }
-      .map{ case (publication, photographers) => publication -> photographers.keys.toList.sortWith(_.toLowerCase < _.toLowerCase) }
+      .groupBy{ case (_, publication) => publication }
+      .map{ case (publication, photographers) => publication -> photographers.map(_._1).sortWith(_.toLowerCase < _.toLowerCase) }
 
-  def creditBylineMap(stores: List[Store]): CreditBylineMap =
-    stores.map(creditBylineMap).reduceLeft(_ |+| _)
```

Comment:
never used (and due to type erasure was ambiguous with its overload now that `Store` is a `List` rather than `Map`)... so removed it